flo-api-generator/src/main/resources/TaskBuilderImpl.mustache

```diff
@@ -92,6 +92,7 @@ final class {{implClassName}} {
     }
 
     @Override
+    @SuppressWarnings("unchecked")
```

Comment:
I like suppressions to have as small scope as possible. Can we put it just on the offending line?